scripts/api-extractor-for-branch.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
```

Comment:
Looks like `api-extractor-for-branch.sh` and `api-extractor.sh` are pretty much the same, except that `api-extractor.sh` always runs on the develop branch.

Would it be possible to have one script and have some sort of parameter to tell it which branch to use?

Reply:
Good idea.